### DISCUSSION TITLE
Project reactor / non-blocking support

### DISCUSSION THREAD

**Discussion by kacperfronc** [jobrunr/jobrunr#262] [Q&A] [answered]
Is there support (or if not, a plan) for reactive frameworks like project reactor, where one could use a reactor loop for scheduling and non-blocking db connection?

I can't find any resources about this

**Comment by rdehuyss** [comment, marked as answer]
Nope, this is not planned. I believe more in project Loom so it was an explicit decision to not do so.

**Reply by kacperfronc** [reply]
I see, thank you
